doc/association-rules.rst

```diff
@@ -0,0 +1,74 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+Data set
```

Comment:
I reckon this is beyond the scope of this pull request. I would opt for a wholesome readjustment later, should we decide to do so for the rest of the documentation.